### ISSUE TITLE
Edit MoBIE Project dialog rows have hardcoded heights

### ISSUE THREAD

**Issue by ctrueden** [mobie/mobie-viewer-fiji#1051]
So on HiDPI screens they are hard to use.
![mobie-hidpi](https://github.com/mobie/mobie-viewer-fiji/assets/556626/654c7664-757f-43e2-96c3-9260b66cf103)

**Comment by tischi** [comment]
Thanks for spotting this! We will trying removing them.

@K-Meech do you have time to do this?

**Comment by K-Meech** [comment]
Yes! I should have time to look at this within the next few weeks.

**Comment by K-Meech** [comment]
I'm looking into this issue now - sorry for the delay! 
@ctrueden - do you only see these issues in the project creator, or also in the main MoBIE ui? E.g. if you run the ```Open Published MoBIE Project...``` command in ImageJ, and select ```Vergara et al...```, is the main panel on the left also affected?

**Comment by ctrueden** [comment]
Yes, the left-side main panel is affected:

![open-published-mobie](https://github.com/mobie/mobie-viewer-fiji/assets/556626/781855fa-ab2b-479b-9421-81777dd42d1e)

I tested with both Zulu OpenJDK 8 and OpenJDK 11 (hopeful that 11's improved HiDPI support would magically fix it, but no dice).

FWIW, the "Edit Mobie Project..." standalone command initial dialog looks fine:

![edit-mobie-project](https://github.com/mobie/mobie-viewer-fiji/assets/556626/c6b76fa4-845e-4604-86df-35bf6ec36b08)

**Comment by K-Meech** [comment]
Thanks for checking @ctrueden . @tischi - it seems then that this is a general issue in MoBIE for all dialogs that create custom JFrames directly (and don't use fiji's default dialogs)

**Comment by ctrueden** [comment]
@K-Meech The problem is caused by [these lines of code](https://github.com/mobie/mobie-viewer-fiji/blob/mobie-viewer-fiji-4.0.5/src/main/java/org/embl/mobie/lib/create/ui/ProjectsCreatorPanel.java#L227-L228) that hardcode pixel sizes.

**Comment by K-Meech** [comment]
Thanks @ctrueden . There are also many places in [SwingHelper](https://github.com/mobie/mobie-viewer-fiji/blob/main/src/main/java/org/embl/mobie/lib/ui/SwingHelper.java) that hardcode pixel sizes for labels + buttons, which probably adds to the issue.
